app/scripts/controllers/preferences.js

```diff
@@ -671,8 +672,8 @@ export default class PreferencesController {
    *
```

Comment:
Nit: the code comment for this method could be slightly modified. `Subscription to NETWORK_DID_CHANGE events` or something like that.

Reply:
reworded comment in the latest commit